### ISSUE TITLE
Experiment settings for some baseline results

### ISSUE THREAD

**Issue by robinliu912** [showlab/data-centric.vlp#5]
Hello,

I appreciate your elegant work. I would like to reproduce some results shown in Table 3.
![image](https://github.com/showlab/datacentric.vlp/assets/77037128/2e37e05d-9915-4efb-9f92-583bc38fc95f)

I try to use openclip (https://github.com/mlfoundations/open_clip/tree/main) to pretrain CLIP with ViT on cc3m but can't get comparable results. Could you please explain the experiment settings in more detail? (e.g.  use openclip or another implementation of clip, encoder choice, learning rate ... etc.)

I look forward to your responce and thank you very much.

sincerely,

**Comment by ZihuaZhao** [comment]
Same question, waiting for reply, much thanks!

**Comment by FingerRec** [comment]
Hi all, the baseline follow blip and vilt. The vision encoder is openai's vit which pretrained on WIT.
